ddtrace/tracer.py

```diff
@@ -187,16 +203,15 @@ def configure(self, enabled=None, hostname=None, port=None, uds_path=None, https
         if isinstance(self.sampler, DatadogSampler):
             self.sampler._priority_sampler = self.priority_sampler
 
-        self._dogstatsd_host = dogstatsd_host or self._dogstatsd_host
-        self._dogstatsd_port = dogstatsd_port or self._dogstatsd_port
-        self.log.debug('Connecting to DogStatsd on {}:{}'.format(
-            self._dogstatsd_host,
-            self._dogstatsd_port,
-        ))
-        self._dogstatsd_client = DogStatsd(
-            host=self._dogstatsd_host,
-            port=self._dogstatsd_port,
-        )
+        if dogstatsd_host is not None and dogstatsd_url is None:
```

Comment:
Then you realize it's not possible to then only pass a port without `dogstatsd_host` being set.

Gosh I hate this method 😠 😆

Reply:
I added a deprecation warning for those parameters, which I agree we should remove.